packages/components/src/Dropdown/Dropdown.tsx

```diff
@@ -249,9 +344,25 @@ function Dropdown(props: DropdownProps) {
     </>
   );
 
-  return <Listbox {...sharedProps}>{childrenToUse}</Listbox>;
+  const contextValue = Object.assign(
```

Comment:
nit: should we re-use this for the case at line 301?

Reply:
Good catch. Thanks Annie!